From 999b53f49d6b0a4c5d5fd123a04686d35118cee0 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 5 Oct 2021 21:02:12 -0700
Subject: [PATCH 19/21] SystemUI: Fix QS status font weight mismatch in dark
 mode

Text in the QS status bar is medium in light mode and regular in dark
mode. Fix the mismatch.

Change-Id: Ic6e5dc547d4ded9d231f88f6cac3e1e9f8483d5e
---
 packages/SystemUI/res/values-night/styles.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/values-night/styles.xml b/packages/SystemUI/res/values-night/styles.xml
index c1d2ca23de31..5f35afb6c731 100644
--- a/packages/SystemUI/res/values-night/styles.xml
+++ b/packages/SystemUI/res/values-night/styles.xml
@@ -37,7 +37,7 @@
     </style>
 
     <style name="TextAppearance.QS.Status" parent="TextAppearance.QS.TileLabel.Secondary">
-        <item name="android:fontFamily">@*android:string/config_bodyFontFamily</item>
+        <item name="android:fontFamily">@*android:string/config_bodyFontFamilyMedium</item>
         <item name="android:textColor">?android:attr/textColorPrimary</item>
     </style>
 
-- 
2.25.1

